client/landing/stepper/declarative-flow/internals/components/step-route/index.tsx

```diff
@@ -6,6 +6,7 @@ import { StepperPerformanceTrackerStop } from 'calypso/landing/stepper/utils/per
 import SignupHeader from 'calypso/signup/signup-header';
 import { useSelector } from 'calypso/state';
 import { isUserLoggedIn } from 'calypso/state/current-user/selectors';
+import SurveyManager from '../survery-manager';
```

Comment:
Typo, "survery-manager" vs. "survey-manager"

Reply:
Oh, I will fix it in another PR.
Thanks!